### DISCUSSION TITLE
我们的项目被做进教程里了诶！

### DISCUSSION THREAD

**Discussion by Mifan-T** [SlopeCraft/SlopeCraft#126] [General｜大堂]
https://www.bilibili.com/video/BV14tBLYWEuV
![image](https://github.com/user-attachments/assets/931556f3-5bea-4221-83a7-caeb47b149ac)

![image](https://github.com/user-attachments/assets/258a5723-7b30-48ee-a30a-f9a25f6a6ba1)

评论区一致好评，就是不知道他们用的是不是新版（）

**Comment by ToKiNoBug** [comment]
好耶！只可惜他们用的是比较旧的版本
